plugin/types.go

```diff
@@ -77,6 +78,9 @@ type ExecOutputChunk struct {
 	Err       error
 }
 
+// StreamTypes provides the name of the corresponding StreamID.
+var StreamTypes = []string{"Stdout", "Stderr"}
```

Comment:
Hmm, think we should just promote `StreamID` to a type and attach a `Name` method. Then callers can do `chunk.StreamID.Name()` to get this info.

Probably worth re-using that logic in https://github.com/puppetlabs/wash/blob/master/api/types/exec.go#L38-L40

Reply:
I'll explore this a bit. Can probably be added on as a separate commit or PR.